src/main/java/core/basesyntax/service/operationhadler/PurchaseHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.operationhadler;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitsTransaction;
+
+public class PurchaseHandler implements TransactionHandler {
+    @Override
+    public void handleTransaction(FruitsTransaction transaction) {
+        int result = FruitStorage.fruitsStorage.get(transaction.getFruit())
+                - transaction.getQuantity();
+        if (result >= 0) {
+            FruitStorage.fruitsStorage.put(transaction.getFruit(), result);
+        } else {
+            throw new RuntimeException("Not enough fruits!");
+        }
```

Comment:
```suggestion
        if (result < 0) {
            throw new RuntimeException("Not enough fruits!");
        }   
        FruitStorage.fruitsStorage.put(transaction.getFruit(), result); 
```
we don't need else after throwing, so we can simplify this a bit

Reply:
not fixed